src/test_power_led.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
```

Comment:
what is the use of this test? As soon as board is powered on , power led glows on.